routes/api.js

```diff
@@ -96,7 +96,7 @@ router.get('/games/:id', async (req, res) => {
 
 		game.frame_url = `${base_url}${game.frame_file}`;
 	} else {
-		game.frame_url = `http://${req.headers.host}/api/files/${game.frame_file}`;
+		game.frame_url = `http${process.env.TLS_CERT ? "s" : ""}://${req.headers.host}/api/files/${game.frame_file}`;
```

Comment:
For this, I'm inclined to use the same protocol as the incoming request instead, just like we use the host value from the incoming request header. That would reduce the reach of the env var.

```suggestion
		game.frame_url = `${req.protocol}://${req.headers.host}/api/files/${game.frame_file}`;
```

See express documentation [here](https://expressjs.com/en/api.html#req.protocol)

Reply:
That looks so much cleaner!